Update README.md (#113329)
dotnet/runtime: docs/workflow/building/libraries/README.md

## APICompat

> If changes to the library include any API incompatibilities, calling `dotnet build` (if the project isn't packable) or `dotnet pack` (if the project is packable) will result in API compatibility errors.

Reviewer: I don't think the (if the project isn't packable) and (if the project is packable) parts are necessary here.
```suggestion
If changes to the library include any API incompatibilities, calling `dotnet build` or `dotnet pack` may result in API compatibility errors.
```

> In rare cases where these are expected (e.g. updating APIs previously shipped only in preview or as experimental), the errors may be suppressed. This can be done by following the directions in the error to re-invoke `dotnet build` (if the project isn't packable) or `dotnet pack` (if the project is packable) with an additional `/p:ApiCompatGenerateSuppressionFile=true` argument.

Reviewer: ```suggestion
In rare cases where these are expected (e.g. updating APIs previously shipped only in preview or as experimental), the errors may be suppressed. This can be done by following the directions in the error to re-invoke the command with an additional `/p:ApiCompatGenerateSuppressionFile=true` argument.
```
?

> In rare cases where these are expected (e.g. updating APIs previously shipped only in preview or as experimental), the errors may be suppressed. This can be done by following the directions in the error to re-invoke `dotnet build` (if the project isn't packable) or `dotnet pack` (if the project is packable) with an additional `/p:ApiCompatGenerateSuppressionFile=true` argument.

Reviewer: ```suggestion
In rare cases where these are expected (e.g. updating APIs previously shipped only in preview or as experimental), the errors may be suppressed. This can be done by following the directions in the error to invoke `dotnet build` (if the project isn't packable) or `dotnet pack` (if the project is packable) with an additional `/p:ApiCompatGenerateSuppressionFile=true` argument.
```

Reviewer: Removing the "re-" I'd added, as the caller may not have used the right command to start.
